packages/tenant-readmodel-writer/src/tenantConsumerServiceV1.ts

```diff
@@ -50,11 +47,9 @@ export async function handleMessage(
           }
         )
     )
-    .with({ type: "SelfcareMappingCreated" }, async (_msg) => {
-      logger.info("TODO");
-    })
-    .with({ type: "SelfcareMappingDeleted" }, async (_msg) => {
-      logger.info("TODO");
-    })
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    .with({ type: "SelfcareMappingCreated" }, async (_msg) => {})
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    .with({ type: "SelfcareMappingDeleted" }, async (_msg) => {})
     .exhaustive();
```

Comment:
aren't `TenantMailAdded` and `TenantMailDeleted` also in v1?
can you check why the exhaustive match isn't failing?

Reply:
This change came in when I merged the main, so it was already present on a previous branch, which is why it wasn't implemented on this one too